rustfft.wai

```diff
@@ -0,0 +1,70 @@
+enum fft-direction {
+    forward,
+    inverse
+}
+
+resource complex {
+    static new-f64: func(re: float64, im: float64) -> complex
+    i: func() -> complex
+}
```

Comment:
We should turn this into a record with `re` and `im` fields. A complex number is just a pair of floats, and if we make it a resource it'll add a lot of copying overhead.

Reply:
Turned the complex into a **record** instead of a **resource**